Orange/widgets/unsupervised/tests/test_owdistances.py

```diff
@@ -40,3 +40,9 @@ def test_error_message(self):
         self.assertTrue(self.widget.Error.no_continuous_features.is_shown())
         self.send_signal("Data", None)
         self.assertFalse(self.widget.Error.no_continuous_features.is_shown())
+
+    def test_mahalanobis_error(self):
+        self.widget.axis = 1
+        self.send_signal("Data", self.iris)
+        self.widget.compute_distances(Mahalanobis, self.iris)
+        self.assertTrue(self.widget.Error.too_few_observations.is_shown())
```

Comment:
Can you please configure PyCharm to always include new line on the last line of file?

http://stackoverflow.com/questions/36043061/automatically-add-newline-on-save-in-pycharm